docs/reference/ilm/getting-started-ilm.asciidoc

```diff
@@ -1,15 +1,227 @@
+{
+  "indices": {
+    "datastream-000001": {
+      "index": "datastream-000001",
+      "managed": true,                           <1>
+      "policy": "datastream_policy",             <2>
+      "lifecycle_date_millis": 1538475653281,
+      "phase": "hot",                            <3>
+      "phase_time_millis": 1538475653317,
+      "action": "rollover",                      <4>
+      "action_time_millis": 1538475653317,
+      "step": "attempt_rollover",                <5>
+      "step_time_millis": 1538475653317,
+      "phase_execution": {
+        "policy": "datastream_policy",
+        "phase_definition": {                    <6>
+          "min_age": "0ms",
+          "actions": {
+            "rollover": {
+              "max_size": "50gb",
+              "max_age": "30d"
+            }
+          }
+        },
+        "version": 1,                            <7>
+        "modified_date_in_millis": 1539609701576
+      }
+    }
+  }
+}
+--------------------------------------------------
+// CONSOLE
+// TESTRESPONSE[s/"lifecycle_date_millis": 1538475653281/"lifecycle_date_millis": $body.indices.datastream-000001.lifecycle_date_millis/]
+// TESTRESPONSE[s/"phase_time_millis": 1538475653317/"phase_time_millis": $body.indices.datastream-000001.phase_time_millis/]
+// TESTRESPONSE[s/"action_time_millis": 1538475653317/"action_time_millis": $body.indices.datastream-000001.action_time_millis/]
+// TESTRESPONSE[s/"step_time_millis": 1538475653317/"step_time_millis": $body.indices.datastream-000001.step_time_millis/]
+// TESTRESPONSE[s/"modified_date_in_millis": 1539609701576/"modified_date_in_millis": $body.indices.datastream-000001.phase_execution.modified_date_in_millis/]
+<1> this index is managed by ILM
+<2> the policy in question, in this case, "datastream_policy"
+<3> what phase the index is currently in
+<4> what action the index is currently on
+<5> what step the index is currently on
+<6> the definition of the phase
+    (in this case, the "hot" phase) that the index is currently on
+<7> the version of the policy being used to execute the current phase
+
+You can read about the full details of this response in the
+<<ilm-explain-lifecycle, explain API docs>>. For now, let's focus on how
+the response details which phase, action, and step we're in. We are in the
+"hot" phase, and "rollover" action. Rollover will continue to be called
+by {ILM} until its conditions are met and it rolls over the index.
+Afterwards, the original index will stay in the hot phase until 90 more
+days pass and it is deleted in the delete phase.
+As time goes on, new indices will be created and deleted.
+With `datastream-000002` being created when the index mets the rollover
+conditions and `datastream-000003` created after that. We will be able
+to search across all of our managed indices using the "datastream" alias,
+and we will be able to write to our to-be-rolled-over write indices using
+that same alias.
+
+=== Next Steps
```

Comment:
Feels strange to have a heading here when it's not really something we want an anchor to. Do you think it's worth keeping? (up to you, I can go either way)